dotnet/src/SemanticKernel/CoreSkills/SemanticFunctionConstants.cs

```diff
@@ -92,15 +91,58 @@ internal static class SemanticFunctionConstants
     inputs:
     - input: the text to email
     - recipient: the recipient's email address. Multiple addresses may be included if separated by ';'.
-[END AVAILABLE FUNCTIONS]
+[END EXAMPLE FUNCTIONS]
 
 <goal>Create an outline for a children's book with 3 chapters about a group of kids in a club and then summarize it.</goal>
 <plan>
   <function._GLOBAL_FUNCTIONS_.NovelOutline input=""A group of kids in a club called 'The Thinking Caps' that solve mysteries and puzzles using their creativity and logic."" chapterCount=""3"" />
   <function.Everything.Summarize/>
 </plan><!-- END -->
 
-End of examples.
+[EXAMPLE FUNCTIONS]
+  Everything.Summarize:
+    description: summarize input text
+    inputs:
+    - input: the text to summarize
+  _GLOBAL_FUNCTIONS_.NovelOutline :
+    description: Outlines the input text as if it were a novel
+    inputs:
+    - input: the title of the novel to outline
+    - chapterCount: the number of chapters to outline
+  LanguageHelpers.TranslateTo:
+    description: translate the input to another language
+    inputs:
+    - input: the text to translate
+    - translate_to_language: the language to translate to
+  EmailConnector.LookupContactEmail:
+    description: looks up the a contact and retrieves their email address
+    inputs:
+    - input: the name to look up
+  EmailConnector.EmailTo:
+    description: email the input text to a recipient
+    inputs:
+    - input: the text to email
+    - recipient: the recipient's email address. Multiple addresses may be included if separated by ';'.
+[END EXAMPLE FUNCTIONS]
+
+<goal>Summarize the input, if the input length is greater than 10 then Create an outline for a children's book with 3 chapters about a group of kids in a club and then summarize it otherwise translate to japanese and email it to Martin.</goal>
+<plan>
```

Comment:
Yowza, this is going to be a token eating machine! We need to drive for simplicity in this format, or it'll burn too many tokens on prompt and response, and the AI is far more likely to get things wrong.

Asks:
at minimum, let's remove <conditiongroup>. Compound conditions can be constructed by nesting one inside another, or with specialized native skills. This is more complex than required right now.
```
 <if>
    <conditiongroup>
       <condition variable=""$SUMMARIZED_INPUT"" greaterthan=""10"" />
    </conditiongroup>
```
...becomes...
` <if variable=""$SUMMARIZED_INPUT"" greaterthan=""10"">`

I also think we can get rid of the `<then>` tags, as these are token eaters and kind of redundant (the content in the `<if...>` tags is implicitly `then`).

As for `else`, I think this could be another conditional stacked below with the inverse condition. I wonder if the AI would do that on its own:

```
<if variable=""$SUMMARIZED_INPUT"" greaterthan=""10"">
    <do stuff>
</if>
<if variable=""$SUMMARIZED_INPUT"" lessthanequals=""10"">
    <do other stuff>
</if>